src/components/Form/Form.tsx

```diff
@@ -116,11 +117,14 @@ export const Form: FC<FormProps> = ({ activity }) => {
     start_form: t('activities.form.cta_start_form'),
   }
 
-  const error_labels = {
+  const error_labels: ErrorLabels = {
     required: t('activities.form.question_required_error'),
     sliderNotTouched: t('activities.form.slider_not_touched_error'),
     invalidPhoneNumber: t('activities.form.invalid_phone_number'),
     formHasErrors: t('activities.form.form_has_errors'),
+    dateCannotBeInTheFuture: t('activities.form.date_cannot_be_in_the_future'),
+    dateCannotBeInThePast: t('activities.form.date_cannot_be_in_the_past'),
+    dateCannotBeToday: t('activities.form.date_cannot_be_today'),
```

Comment:
@mohsinht Shouldn't these be added to the translation files or are these now managed via Localazy? In any case, might be good to post the English copy  in `#product-feedback` and ask for Hafsa's input.

Reply:
Yes these should be updated using Localazy. I've manually added the translations for now but those need to be added in Localazy.